packages/components-js/src/pages/tag-dismissible.html

```diff
@@ -39,6 +61,19 @@
   <p-tag-dismissible label="Some label" color="background-surface">Color background-surface</p-tag-dismissible>
 </div>
 
+<div
+  class="playground dark surface"
+  title="should show different background colors and label on dark surface background"
+>
+  <p-tag-dismissible theme="dark" label="Some label">Default</p-tag-dismissible>
+  <p-tag-dismissible theme="dark" label="Some label" color="background-default"
+    >Color background-default</p-tag-dismissible
+  >
+  <p-tag-dismissible theme="dark" label="Some label" color="background-surface"
+    >Color background-surface</p-tag-dismissible
+  >
+</div>
+
```

Comment:
isn't a test scenario missing for `should show different multiline tags on light background`?

Reply:
multiline tags have no impact on the theme? We want to ensure the position of the `X` is correct when it is multiline.